src/styles/SelectedRevsStyle.ts

```diff
@@ -52,19 +52,17 @@ export const SelectRevsStyles = (mode: string) => {
       },
     },
 
-    listItemButton: {
-      cursor: 'auto !important',
-      padding: '0 !important',
-      alignItems: 'flex-start !important',
+    selectedRevision: {
+      display: 'flex',
+      cursor: 'auto',
+      padding: '0',
+      alignItems: 'flex-start',
+      minWidth: '0px',
       $nest: {
         '&:hover': {
-          backgroundColor: 'transparent !important',
+          backgroundColor: 'transparent',
```

Comment:
I think it's not needed either, this was also used to override the button's css properties.